examples/variational_inference/empirical-approx-overview.ipynb

```diff
@@ -18,11 +18,13 @@
      "name": "stdout",
```

Comment:
<p>Missing output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Done! </p><p> </p><p>Note:</p><p>There was an update for the <code>sns.kdeplot </code>plot functionality, and created an error.</p><p> </p><p>I went ahead and provided the same visualization by creating the 100x2 array into a DataFrame, and specified index in <code>kdeplot</code> params.</p>